docs/sources/setup/size/_index.md

```diff
@@ -17,72 +17,70 @@ weight: 100
 <!-- vale Grafana.Quotes = NO -->
 <!-- vale Grafana.Quotes = YES -->
 
-This tool helps to generate a Helm Charts `values.yaml` file based on specified
- expected ingestion, retention rate and node type. It will always configure a
- [scalable]({{< relref "../../get-started/deployment-modes#simple-scalable" >}}) deployment. The storage needs to be configured after generation.
+This section is a guide to size base resource needs of a Loki cluster.
```

Comment:
I can't select the right line as a comment but lines 2 - 4 will need to be removed so it appears in the docs
```
_build:
  list: false
noindex: true
```